llvm/test/tools/llvm-profdata/memprof-basic.test

```diff
@@ -8,7 +8,7 @@ additional allocations which do not originate from the main binary are pruned.
 
 CHECK:  MemprofProfile:
 CHECK-NEXT:   Summary:
-CHECK-NEXT:     Version: 4
+CHECK-NEXT:     Version: 5
```

Comment:
Do we still have any v4 tests? Updating these other tests seems fine but would be good to make sure we still have one for the old version(s)

Reply:
Done. Forked the existing basic test to a basic_v4 test like we have for v3.